src/Antiforgery/test/Directory.Build.props

```diff
@@ -1,10 +1,3 @@
 <Project>
```

Comment:
Suggest removing this file. There's no `Directory.Build.props` in lower-level folders. So, the file in the pseudo-repo root (`src/Antiforgery`) will be found automatically.